third_party/txt/tests/paragraph_unittests.cc

```diff
@@ -3568,4 +3568,49 @@ TEST_F(ParagraphTest, DISABLE_ON_WINDOWS(StrutForceParagraph)) {
   ASSERT_TRUE(Snapshot());
 }
 
+TEST_F(ParagraphTest, FontFeaturesParagraph) {
+  const char* text = "123456789\n";
```

Comment:
Should probably check against non-numeric glyphs being unaffected by tnum as well as the what the numbers without this feature layout as.